snap/validate_test.go

```diff
@@ -253,6 +253,9 @@ func (s *ValidateSuite) TestValidateAppSocketsInvalidListenStreamPathRelative(c
 func (s *ValidateSuite) TestValidateAppSocketsInvalidListenStreamAbstractSocket(c *C) {
 	app := createSampleApp()
 	invalidListenAddresses := []string{
+		"@snap.myapp",
+		"@snap.myapp.foo",
+		"@snap.myapp\\000.foo",
```

Comment:
I was thinking that 'myapp' was the the snap name, but it is 'mysnap'. Can you remove '@snap.myapp.foo' since it is covered below, then change the other two to be:
```
"@snap.mysnap",
"@snap.mysnap\\000.foo",
```

Reply:
done